### ISSUE TITLE
[beta] Tests never finishes running in Edge

### ISSUE THREAD

**Issue by descadora** [freeCodeCamp/freeCodeCamp#13473]
http://beta.freecodecamp.com/en/challenges/applied-visual-design/create-visual-balance-using-the-textalign-property

I have added what I believe to be the correct code to the challenge, and after reloading the screen the images show up within the phone and even adjusts as I add to the code.  However, it never progresses beyond the phrase "running test", and therefore I cannot pass it.  Forgive me if this is the same as the other issues posted, I do see that you guys are having a lot of problems with this and I very much appreciate the hard work in fixing them.  Thanks for bringing us the new content, too.

* Browser Name, Version: Microsoft Edge, version unknown 
* Operating System: Windows 10 Home, version 1607
* Mobile, Desktop, or Tablet: Laptop

![fcc bug screenshot](https://cloud.githubusercontent.com/assets/24569208/23137790/b2c85f7c-f769-11e6-9988-fc9f5fc43324.PNG)

**Comment by Greenheart** [comment]
@descadora Thanks for reporting this!

I can pass the challenge with your code in Chrome 56.
In Firefox 51, the rests finish running, but I somehow can't complete the challenge despite using the exact same code as I did n Chrome.

Can you see if there's any error messages in your browser console. I'm not sure how to open the developer tools in Edge, but it's f12 in both chrome and firefox.

Also, please try to 1. copy your solution, 2. do a hard refresh (CTRL+SHIFT+R), 3. using the **Reset** button and 4. trying to re-submit your solution.

**Comment by descadora** [comment]
Thanks for the answer. Opening dev tools in Edge is done with a
right-click. However, I tried to switch to using IE instead and now in both
Edge and I.E. the stuff in the phone won't load, and I get a repeating
vertical white line running through the code area and the test area below
the buttons. It's kind of a fascinating bug, but not very helpful. I was
also, on first attempt, getting pop-ups that were blank but for a hash
symbol. I'm ridiculously curious how any of these things are happening.
Anyway, error messages: none that I can find.
Hope this helps?
Thanks!

**Comment by Greenheart** [comment]
@descadora Thanks for taking the time! :tada: 

The continuously running line is a common issue that I've seen several times the past weeks. It's most common in Firefox but IE/Edge seem to have the same issue as well. 

The other issues had error messages in the console saying that the react bundle for the page had crashed. So I assume it's something similar.

In an ideal world, things would work the same across all browsers (and things are moving in that direction thankfully), but sometimes there's not more to do than to try a different browser. Would you mind switching to Chrome temporarily and see if that at least could allow you to keep learning while we fix this? :smile:

**Comment by descadora** [comment]
If my connection will allow, I will see about downloading it tomorrow.
Thanks so much, for the help and the explanation. Have a great day!

**Comment by Greenheart** [comment]
@descadora Thanks, you too! :blush:

Good luck, and happy coding!

**Comment by systimotic** [comment]
Firefox finishes running, but doesn't pass because of #10749 (I've referenced this issue so many times now that I remember the first three numbers 😅)

The error:
```
SCRIPT5007: Unable to get property 'helpers' of undefined or null reference
create-visual-balance-using-the-textalign-property (53,413)
```

`SCRIPT5007` links to [this MSDN page](https://msdn.microsoft.com/query/dev12.query?appId=Dev12IDEF1&l=EN-US&k=k(VS.WebClient.Help.SCRIPT5007))
Clicking the bottom line brings me to the end of the HTML file, that doesn't even have 53 lines.
I checked the source of the iframes, and found what it references. There's a script in the iframe after the user's code. This refers to what in the source code is `client\frame-runner.js` line 7.

Perhaps closer to the cause of the issue: `document.Rx` is `undefined` in Edge. It is defined in Chrome, inside the iframe. I haven't found where it is defined, and why it is defined in Chrome but not in Edge.

This occurs for every challenge I tested in Edge.
